docs/concepts/services-networking/service.md

```diff
@@ -428,30 +428,26 @@ an ephemeral IP will be assigned to the loadBalancer. If the `loadBalancerIP` is
 cloud provider does not support the feature, the field will be ignored.
 
 #### SSL support on AWS
-For partial SSL support on clusters running on AWS, starting with 1.3 two
-annotations can be added to a `LoadBalancer` service:
+For partial SSL support on clusters running on AWS, the following annotations can be 
+added to a `LoadBalancer` service:
 
-```
+```yaml
     metadata:
       name: my-service
       annotations:
         service.beta.kubernetes.io/aws-load-balancer-ssl-cert: arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012
+        service.beta.kubernetes.io/aws-load-balancer-backend-protocol: (https|http|ssl|tcp)
+        service.beta.kubernetes.io/aws-load-balancer-ssl-ports: '443'
 ```
 
-The first specifies which certificate to use. It can be either a
-certificate from a third party issuer that was uploaded to IAM or one created
-within AWS Certificate Manager.
+`aws-load-balancer-ssl-cert` (added in version 1.3) specifies which certificate to use.
+It can be either a third party certificate uploaded to IAM, or one created within AWS Certificate Manager.
 
-```yaml
-    metadata:
-      name: my-service
-      annotations:
-         service.beta.kubernetes.io/aws-load-balancer-backend-protocol: (https|http|ssl|tcp)
-```
+`aws-load-balancer-backend-protocol` (added in version 1.3) specifies which protocol a pod speaks. For HTTPS and
+SSL, the ELB will expect the pod to authenticate itself over the encrypted connection.
```

Comment:
Checked the AWS documentation - for HTTPS the pod will need its own certificate. You are then responsible for adding the public key of the certificate to the AWS ELB.